nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -303,13 +383,21 @@ geometry_msgs::msg::TwistStamped RegulatedPurePursuitController::computeVelocity
 
   linear_vel = desired_linear_vel_;
 
+  distance_profile_input_.max_velocity = {desired_linear_vel_};
```

Comment:
Isn't this already set?

Reply:
Yes, but logic is as follows:

1. Set maximum velocity to `desired_linear_vel_`
2. RPP algorithm modifies maximum velocity in order to folow path

Because maximum velocity limit in motion profile is changed, we must reload it in step 1.

Reply:
This is another place where semantically meaningful functions and comments are important